transforms/helpers/ast.ts

```diff
@@ -228,26 +219,27 @@ export function isEOPropertyForClassDecorator(
   );
 }
 
-export type EOClassDecoratorValue = Literal | ArrayExpression | Identifier;
+export type EOClassDecoratorValue =
+  | StringLiteral
+  | ArrayExpression
+  | Identifier;
 
-export function isEOClassDecoratorValue(
-  u: unknown
-): u is EOClassDecoratorValue {
+function isEOClassDecoratorValue(u: unknown): u is EOClassDecoratorValue {
   return (
-    isNode(u, 'Literal') ||
+    isNode(u, 'StringLiteral') ||
     isNode(u, 'ArrayExpression') ||
     isNode(u, 'Identifier')
   );
 }
 
-type EOClassDecoratorKey = Identifier & {
+interface EOClassDecoratorKey extends Identifier {
   name:
     | 'layout'
```

Comment:
I'd do this in `util/index.ts`

```ts
export type LAYOUT_DECORATOR_NAME = typeof LAYOUT_DECORATOR_NAME.
```

and then

```suggestion
    | LAYOUT_DECORATOR_NAME
```

Reply:
Neat.